multinode-demo/common.sh

```diff
@@ -42,7 +42,8 @@ else
     if [[ -n $NDEBUG ]]; then
       maybe_release=--release
     fi
-    printf "cargo run $maybe_release $maybe_package --bin solana-%s %s -- " "$program" "$features"
+    declare manifest_path="--manifest-path=$program/Cargo.toml"
```

Comment:
Should $program have quotes around it?

Reply:
Maybe. Works fine for now since all the callers of this func are defined below and were simple enough. But I guess putting quotes around it is much more extensible.

Reply:
I'm surprised that shellcheck didn't flag it.

Reply:
The option is already in quotes. Do you need to quote the variables within too?